rfc077-mode-agnostic-data-collection.md

```diff
@@ -525,6 +445,7 @@ The Run End Schema will be used by Chef Client to notify the data collection ser
         "expanded_run_list",
         "message_type",
         "message_version",
+        "node",
```

Comment:
I'm still thinking about this, but do we really want to require a full node object in this message?